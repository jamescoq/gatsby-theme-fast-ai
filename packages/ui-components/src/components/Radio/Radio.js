import React from 'react';

import RadioControl from '../RadioControl';
import FormField from '../FormField';

const Radio = props => <FormField control={RadioControl} kind="radio" {...props} />;

export default Radio;
