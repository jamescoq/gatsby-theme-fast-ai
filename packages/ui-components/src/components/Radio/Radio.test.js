import React from 'react';
import { shallow } from 'enzyme';

import Radio from './Radio';

describe('<Radio />', () => {
	const requiredProps = {};

	it('shallow', () => {
		const wrapper = shallow(<Radio {...requiredProps} />);
		expect(wrapper).toMatchSnapshot();
	});
});
