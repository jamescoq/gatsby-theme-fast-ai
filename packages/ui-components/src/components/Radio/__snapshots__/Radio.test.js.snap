// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Radio /> shallow 1`] = `
<withGeneratedId(forwardRef(FormField))
  control={[Function]}
  kind="radio"
/>
`;
